hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/s3/tenant/OMTenantRequestHelper.java

```diff
@@ -99,26 +100,29 @@ static void checkTenantExistence(OMMetadataManager omMetadataManager,
    * Retrieve volume name of the tenant.
    */
   static String getTenantVolumeName(OMMetadataManager omMetadataManager,
-      String tenantId) {
+      String tenantId) throws OMException {
 
     final OmDBTenantInfo tenantInfo;
     try {
       tenantInfo = omMetadataManager.getTenantStateTable().get(tenantId);
```

Comment:
We don't want this one in a try/catch. An exception here does not mean the tenant was not found, it means there was an error accessing RocksDB. This should terminate the OM. OzoneManagerRatisUtils#exceptionToResponseState will read the IOException to determine whether it warrants an error response to the user or OM termination. This means this method should throw IOException. Error handling in the two cases below looks good though.

Reply:
Makes sense. done